enterprise/migrations/0001_initial.py

```diff
@@ -0,0 +1,125 @@
+                                   null=True, upload_to='enterprise_logos')),
+                ('data_sharing_policy', models.CharField(
+                    choices=[('no_sharing', 'No data sharing requested'), ('request', 'Request data sharing'),
+                             ('require', 'Require data sharing')], help_text='Data sharing policy', max_length=20)),
+                ('active', models.BooleanField(default=True)),
+            ],
+            options={
+                'verbose_name': 'Enterprise Customer',
+                'verbose_name_plural': 'Enterprise Customers',
+            },
+        ),
+        migrations.CreateModel(
+            name='EnterpriseCustomerHistory',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created', model_utils.fields.AutoCreatedField(default=django.utils.timezone.now, editable=False,
+                                                                verbose_name='created')),
+                ('modified', model_utils.fields.AutoLastModifiedField(default=django.utils.timezone.now, editable=False,
+                                                                      verbose_name='modified')),
+                ('name', models.CharField(help_text='Enterprise Customer name.', max_length=500)),
+                ('short_code', models.CharField(
+                    help_text='Short code will be used to reference this Enterprise Customer in other parts of the system,i.e. SSO, ecommerce app, analytics etc.',
+                    max_length=120)),
+                ('logo',
+                 models.ImageField(blank=True, help_text='Please add only .PNG files for logo images.', max_length=255,
+                                   null=True, upload_to='enterprise_logos')),
+                ('data_sharing_policy', models.CharField(
+                    choices=[('no_sharing', 'No data sharing requested'), ('request', 'Request data sharing'),
+                             ('require', 'Require data sharing')], help_text='Data sharing policy', max_length=20)),
+                ('active', models.BooleanField(default=True)),
+                ('origin', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='history',
+                                             to='enterprise.EnterpriseCustomer')),
+            ],
+            options={
+                'verbose_name': 'Enterprise Customer History',
+                'verbose_name_plural': 'Enterprise Customer Histories',
+            },
+        ),
+        migrations.CreateModel(
+            name='EnterpriseCustomerMember',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('user_id', models.IntegerField()),
+                ('enterprise_customer',
```

Comment:
I think this comes from the field name, see: https://github.com/edx/edx-enterprise/pull/1/files#diff-4bcae564bc272f5c2e6753dc6021eb1cR93